projectcharter.md

```diff
@@ -0,0 +1,84 @@
+  
+  
+## Stakeholders
+#### Who needs to be informed about progress in major decisions?
+* Sponsor - Suzette King (OFCIO)
+* USDS Leadership
+* DHS Digital Service team
+* Others?
+
+## Project communication expectations
+**What is the best method for staying in touch?**
+* Hipchat - Paperless.gov channel
+* GitHub - us-forms-system repo
+* Email
+* Others?
+
+## What should status reports include?
+Progress towards project goals, blockers
+
+## Key Players
+* Julie Meloni | Project Lead
+* Ellen Butters | UX Designer
+* Anne Krainic | Engineer
+* Ju-Lie McReynolds | Product Manager
+* Others?
+
+## Project goal
+#### What will be the end result of the project?  
+A foundation for optimizing the process of building more useful, modern, and frictionless web-based forms through the creation of a public open-source forms library that will incorporate the U.S. Web Design Standards and include guidance and best practices for use.
+
+## Scope
+#### What is in scope?
+1. The aggregation and public sharing of lessons-learned from multiple years of USDS projects involving the transformation of forms from paper to web-based.
+2. The creation and public open-source release of a forms library based on the DSVA extensions to RSJF. This library could be used government-wide, and will incorporate the U.S. Web Design Standards along with guidance for data storage and integration best practices.
+3. The use of this library in production by at least one agency (besides VA).
+4. The creation of exemplars for conversion of high-value paper-based forms to web-based forms using this library.
+
+#### What is not in scope?
+* Support for additional form builders or rendering non-React components
+* Integration with other government digital services (e.g. login.gov)
+* Others?
+
+## Milestones
+#### …including deadlines
+* Worthy thing 1 - with date
+* Worthy thing 2 - with date
+
+## Dependencies
+#### Are there other projects/people upon whom the success of this project relies?
+* Non-team person 1
+* Other team 1
+* Other team 2
+
+## Risks to completion
+#### What are the potential risk factors that could keep us from meeting the project goal(s)?
+* Risk 1
+* Risk 2  
+
+#### What do we assume we can rely on to achieve the project goal(s)?
+* Executive support (?)
+* Team time commitments (?)
+  * Both Anne and Ju-Lie will remain 1x allocated for the full period of this project
+  * Ellen will remain available at .5x for this project
+  * Julie will continue to have continue to have .2x availability to contribute
+* Contracted resources will be available to support the engineering efforts
+
+## Exit criteria
```

Comment:
This is interesting. This framework sort of assumes that there's a finality to the project, that it can end with a specific goal being achieved. However, I don't see this work ever really ending. Since we'll be creating an open source library, it will have to be maintained over time, new features will probably need to be added, bugs/issues fixed. I think our exit criteria is rather figuring out a long-term maintenance plan and transferring responsibility to those people.

Reply:
I included some additional explanation in the Exit Plan section. 

I've been thinking of this particular charter as being tightly constrained to the initial scope of work through the end of September, and any additional on-going/follow-on work would be picked up in a separate document. This could very well not be the best way to approach this. I'm definitely open to shifting to another framing.